aws-ecsfargate-terraform/README.md

```diff
@@ -1,23 +1,29 @@
-# Deploying the 1Password SCIM Bridge on AWS Fargate with Terraform
+# Deploy 1Password SCIM Bridge on AWS Fargate with Terraform
 
-This guide will run you through a deployment of the 1Password SCIM bridge on AWS Fargate using Terraform. 
+*Learn how to deploy 1Password SCIM Bridge on AWS Fargate using Terraform.*
 
-Note that due to the highly advanced and customizable nature of Amazon Web Services, this is only a suggested starting point. You may modify it to your needs to fit within your existing infrastructure.
+> **Note**
+> 
+> Due to the highly advanced and customizable nature of Amazon Web Services, this is only a suggested starting point. You can modify it to fit your existing infrastructure.
 
-## Prerequisites
+**Table of contents:**
 
-Before beginning, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
+- [Before you begin](#before-you-begin)
+- [Step 1: Configure the deployment](#step-1-configure-the-deployment)
+- [Step 2: Deploy 1Password SCIM Bridge](#step-2-deploy-1password-scim-bridge)
+- [Step 3: Connect your identity provider](#step-3-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Troubleshooting](#troubleshooting)
 
-- Install [Terraform](https://www.terraform.io/downloads)
-- Have your `scimsession` file and bearer token (as seen in `PREPARATION.md`) ready
+## Before you begin
 
-## Sign in with `aws`
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there. Then:
 
-Ensure you are authenticated with the `aws` tool in your local environment.
-
-See [Terraform AWS Authentication](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication) for more details.
+- Install [Terraform](https://www.terraform.io/downloads)
+- Have your `scimsession` file and bearer token (as seen in `PREPARATION.md`) ready
+- Make sure you're authenticated with the `aws` command-line tool in your local environment. Learn more in the [Terraform AWS Provider documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs).
 
-## Configuration
+## Step 1: Configure the deployment
 
 ### Copy configuration template
```

Comment:
I feel like the steps in this section are calling out for numbers. Any reason not to make these an ordered list?